src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -312,6 +340,37 @@ private Response invokeCallbackEndpoint(
             .post("/callback/attach_case?ignore-warning=true");
     }
 
+    private Response invokeCallbackEndpointWithSearchCaseRefAndAttachToCaseRef(
+        CaseDetails searchCaseRefCaseDetails,
```

Comment:
do you need to pass whole object here ? just case id is used so may be just pass that ?